CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### Fixed
 - Undetermined treatment of a client without the client id specified.
 - Provisioning of a client with service account enabled when the `registrationEmailAsUsername` flag for the realm is set to `true`.
+- 409 Conflict on importing client role that already exists but not in state.
```

Comment:
Since 3.3.0 is already released, please put this line to Unreleased.

Reply:
Oh, I didn't notice that when resolving merge conflicts on this file. Moved this line to Unreleased